woocommerce/woocommerce
# Recurring payment badge on the gateways list

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This adds a "Recurring payments" icon next to providers offering that.





Closes #53384



### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

**Pre-requisites for local testing:**

- Have woocommerce installed locally and synced within WooPayments. See Pe4Hk0-3f-p2 for details.
- Enable the feature flag to see reactified settings page. pe4Hk0-1ly-p2.
- Check out this branch.
- Run pnpm i && pnpm run build



**Testing instructions:**

1. First, you need **WooCommerce Subscriptions** to test this. Please get the zip and install it locally.
1. Navigate to WooCommerce > Settings > Payments
2. Select a country where WooPayments is available (UAE for me)
3. You should see a new "Recurring payments" icon next to WooPayments

<img width="1458" alt="Screenshot 2024-12-12 at 16 25 34" src="https://github.com/user-attachments/assets/cb52584e-1994-4f65-92b0-106d27d5d72b" />

4. On hovering, a tooltip should be displayed:

<img width="644" alt="Screenshot 2024-12-12 at 16 33 02" src="https://github.com/user-attachments/assets/aeb9a384-0818-498e-b7d6-60e52351243e" />

5. Go to "Other payments options" and install Stripe
6. Once installed, the same icon and tooltip should be displayed too

<img width="739" alt="Screenshot 2024-12-12 at 16 35 09" src="https://github.com/user-attachments/assets/9d7104d5-958b-4f9a-9096-618c078de007" />



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Add "Recurring payments" icon next to payment providers

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
